abstract.txt

```diff
@@ -0,0 +1,23 @@
+Open data formats and software for gamma-ray astronomy
+======================================================
+
+Authors: tbd
+
+In gamma-ray astronomy, a lot of proprietary data formats and software exist
+that were developed for one specific mission or experiment. Especially in
```

Comment:
In gamma-ray astronomy, a variety of data formats and proprietary software exist,
 developed for one specific mission or experiment.